crates/miden-objects/src/note/note_tag.rs

```diff
@@ -459,47 +444,13 @@ mod tests {
             | 0x00cc_77cc_0000_0000_0000_0000_0000_0000;
         let network_account_id = AccountId::try_from(NETWORK_ACCOUNT_INT).unwrap();
 
-        // Expected network tag with LOCAL_EXECUTION_WITH_ALL_NOTE_TYPES_ALLOWED.
-        let expected_network_local_tag = NoteTag(0b11101010_10110011_00000000_00000000);
```

Comment:
nit: I would probably move the declaration from 427 down here so that we declare the variable closer to the place where we use it.